### PR TITLE
CLOUDP-85231: No path hacking needed with go1.16

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-85231

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

With go1.16 we no longer need to be hopping dirs t avoid unintended changes to the go.mod file

### CODE REVIEW

**Review by andreaangiolillo — DISMISSED** [review]
LGTM if LGTEV

**Comment by gssbzn** [comment]
EVG doesn't like it, it looks like `go install github.com/google/addlicense@latest` is failing for some reason

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -24,11 +24,22 @@ export PATH := ./bin:$(PATH)
...
 
-.PHONY: setup
-setup:  ## Install dev tools
-	@echo "==> Installing dependencies..."
+.PHONY: setupgolangcilint
+setupgolangcilint:  ## Install golangci-lint
+	@echo "==> Installing golangci-lint..."
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_VERSION)
 
+.PHONY: deps
```
Comment (gssbzn) [inline Makefile:32]: FYI @shalka I took some of your changes from #542 since I was here

**Pull request merged**